EXPLAINER.md

```diff
@@ -149,6 +149,22 @@ document.querySelector("#abortButton").onclick = event => {
 };
 ```
 
+## Integration with Page Visibility
+
+Web NFC functionality is allowed only for the Document of the top-level browsing
```

Comment:
Since we are not linking to specs, I think it is fine to write document (lowercase)

Reply:
Done